INSTALL.md

```diff
@@ -73,18 +73,18 @@ This will recreate out-of-date files in `./dist`, create a folder
 `$APACHE_BASEURL` in `$APACHE_DIR` and copy all the files from `./dist` to
 `$APACHE_DIR/$APACHE_BASEURL` using `sudo` with user `$APACHE_USER`.
 
-Deployment can be customized with three environment variables, the default is:
+Deployment can be customized with four environment variables, the default is:
 
 ```
-make APACHE_USER=www-default APACHE_DIR=/var/www/html APACHE_BASEURL=ocr-gt-tools deploy
+make APACHE_USER=www-data APACHE_GROUP=www-data APACHE_DIR=/var/www/html APACHE_BASEURL=ocr-gt deploy
```

Comment:
Looks good. @kba, @zuphilip, `ocr-gt` is the current default for the base URL. Is this what we want, or should it be fixed to be `ocr-gt-tools`?